core/contracts/oracle/implementation/Voting.sol

```diff
@@ -358,6 +358,8 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface {
      * @notice Reveal a previously committed vote for `identifier` at `time`.
      * @dev The revealed `price`, `salt`, `time`, `address`, `roundId`, and `identifier`, must hash to the latest `hash`
      * that `commitVote()` was called with. Only the committer can reveal their vote.
+     * @dev Since transaction data is public, the salt will be revealed with the vote. While this is the system’s expected behavior,
+     * it assumes that voters will never reuse the same pair of salt and price. Otherwise their commits could be easily disclosed in advance.
```

Comment:
I think we should rephrase this slightly to discourage voters from reusing the same salt twice, not just salt-price pair. Since an attacker may have some ability to predict the price for a particular vote, so a known salt and an unknown price wouldn't offer that much protection.

Reply:
done